<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>615</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>我的音乐</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/image/ios8_Music_32px_1173465_easyicon.net.png</normaloff>:/image/image/ios8_Music_32px_1173465_easyicon.net.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>5</number>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Widget
{

    border:1px solid #5F5F5F;
	
	border-radius:0px;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>5</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-image: url(:/image/image/add_32px_1163593_easyicon.net.png);
background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>560</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
    QSlider::add-page:Horizontal
     {     
background-color: rgba(255, 255, 255,100);
        height:4px;  
     } 
     QSlider::sub-page:Horizontal   
    {  
background-color: rgb(255, 255, 255);
        height:4px;
     }
    QSlider::groove:Horizontal   
    { 
        background:transparent; 
        height:4px;
    }  
    QSlider::handle:Horizontal  
    { 
        height:16px; 
        width:16px; 
	
	border-image: url(:/image/image/圆形 (9).png);
  margin: -6 0px; 
    }
      
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>550</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 11pt &quot;Courier&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>550</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{


	background-image: url(:/icons/icons/next.png);


background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}
QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

	background-image: url(:/icons/icons/last.png);


background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>570</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{


background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>570</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{



background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{

	background-image: url(:/image/image/list_32px_1074296_easyicon.net.png);

background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>50</y>
     <width>300</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_3"/>
   <widget class="QWidget" name="page_2">
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>300</width>
       <height>500</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget
{
background-color:transparent;
color: rgb(0, 0, 0);
font: 75 11pt &quot;Comic Sans MS&quot;;


}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QListWidget" name="listWidget_4">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>300</width>
       <height>500</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget
{
background-color:transparent;
color: rgb(0, 0, 0);
font: 75 11pt &quot;Comic Sans MS&quot;;


}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>530</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Franklin Gothic Book&quot;;
color: rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>5</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-image: url(:/image/image/最小化 (1).png);
background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}

QPushButton:hover{



	background-color: rgb(255, 255, 255,100);


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>5</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-image: url(:/image/image/叉号.png);
background-color: rgba(255, 255, 255,0);
border-style:outset;
border-color:rgba(255,255,255,0);
border-radius:4px;

}

QPushButton:hover{


background-color: rgba(255, 255, 255,100);
	;


}
QPushButton:pressed{
	background-color: rgba(95, 95, 95,100);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>400</y>
     <width>35</width>
     <height>161</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> QSlider 
    {  
        background-color: rgba(20, 20, 20,200);
        min-width:5px; 
        max-width:5px;
        border:15px solid rgba(25, 38, 58,100); 
    }
     QSlider::add-page:vertical
     {      
       
	background-color: rgb(250, 250, 0);
        width:4px;
     }
     QSlider::sub-page:vertical  
    {
        background-color: rgb(87, 97, 106); 
        width:4px;
     }
    QSlider::groove:vertical 
    {
        background:transparent;
        width:6px;
    }
    QSlider::handle:vertical 
    {
         height: 13px;
        width:13px;
 
 margin: -0 -4px; 
	border-image: url(:/image/image/圆形 (8).png);
     }</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>verticalSlider</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
